app/src/main/kotlin/Reversi/ReversiSolverMinimax.kt

```diff
@@ -4,38 +4,91 @@ import mcts.StateNode
 import mcts.StatelessActionNode
 import java.awt.Point
 
-class ReversiSolverMinimax(initialState: ReversiState)
+class ReversiSolverHeuristicSim(initialState: ReversiState)
```

Comment:
@JunTaoLuo is this file still called `ReversiSolverMinimax.kt`

Reply:
Yea it should be renamed.